.github/workflows/dev.yml

```diff
@@ -0,0 +1,33 @@
+name: Dev branch CI
+
+on:
+  pull_request:
+    branches:
+      - dev
```

Comment:
Might be worth setting up a CI for the main branch too.

Reply:
Updated the file to inclues the main branch.